code/src/components/Styling.js

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components'
+
+export const ButtonSwitch = styled.button`
```

Comment:
Great use of styled components, I haven't seen media queries embedded in these before!